packages/manageiq-release/manageiq-release.spec

```diff
@@ -36,6 +36,9 @@ rm -rf $RPM_BUILD_ROOT
 /etc/pki/rpm-gpg/*
 
 %changelog
+* Tue Nov 9 2021 Jason Frey <fryguy9@gmail.com> - 13.0-1%{dist}
```

Comment:
This should probably match the release date that's on master for 13.0-1 https://github.com/ManageIQ/manageiq-rpm_build/blob/a36bd06b3a28fdea2b8403aea21f659ea208693e/packages/manageiq-release/manageiq-release.spec#L45

Reply:
Done